packages/recommend/src/__tests__/getTrendingItemsForFacet.test.ts

```diff
@@ -0,0 +1,48 @@
+import { TestSuite } from '../../../client-common/src/__tests__/TestSuite';
+
+const recommend = new TestSuite('recommend').recommend;
+
+function createMockedClient() {
+  const client = recommend('appId', 'apiKey');
+  jest.spyOn(client.transporter, 'read').mockImplementation(() => Promise.resolve());
+
+  return client;
+}
+
+describe('getTrendingItemsForFacet', () => {
+  test('builds the request', async () => {
+    const client = createMockedClient();
+
+    await client.getTrendingItemsForFacet(
+      [
+        {
+          indexName: 'products',
+          facetName: 'company',
+          facetValue: 'tesla',
+        },
+      ],
+      {}
+    );
+
+    expect(client.transporter.read).toHaveBeenCalledTimes(1);
+    expect(client.transporter.read).toHaveBeenCalledWith(
+      {
+        cacheable: true,
+        data: {
+          requests: [
+            {
+              indexName: 'products',
+              model: 'trending-items',
+              facetName: 'company',
+              facetValue: 'tesla',
+              threshold: 0,
+            },
+          ],
+        },
+        method: 'POST',
+        path: '1/indexes/*/recommendations',
+      },
+      {}
+    );
+  });
+});
```

Comment:
I add issue with Docker, and couldn't run the test suit on my side. But I guess it runs while building the PR?